components/leaderboard/RankingsTable.tsx

```diff
@@ -42,7 +43,7 @@ const RankingsTable: FunctionComponent<RankingProps> = ({
           const completedQuestsResponse = await getCompletedQuests(
             item?.address
           );
-          item.completedQuests = completedQuestsResponse?.length;
+          item.completedQuests = (completedQuestsResponse as CompletedQuests)?.length;
```

Comment:
we already return the response in this format no? why are we mentioning it again here?

Reply:
The function returns `CompletedQuest | QueryError`. I am trying to get only `CompletedQuest` because that's where I can access the `length` property

Reply:
got it